draft-ietf-httpbis-sfbis.md

```diff
@@ -464,6 +465,22 @@ For example:
 Example-Date: @1659578233
 ~~~
 
+### Display Strings {#displaystring}
+
+Display Strings are similar to Strings, in that they consist of zero or more characters, but they allow unicode content, unlike Strings.
+
+Display Strings are intended for use in cases where a value is displayed to end users, and therefore may need to carry non-ASCII content. It is NOT RECOMMENDED that they be used in situations where a String ({{string}}) or Token ({{token}}) would be adequate, because unicode has processing considerations (e.g., normalization) and security considerations (e.g., homograph attacks) that make it more difficult to handle correctly.
```

Comment:
```suggestion
Display Strings are intended for use in cases where a value is displayed to end users, and therefore may need to carry non-ASCII content. It is NOT RECOMMENDED that they be used in situations where a String ({{string}}) or Token ({{token}}) would be adequate, because Unicode has processing considerations (e.g., normalization) and security considerations (e.g., homograph attacks) that make it more difficult to handle correctly.
```